common/scripts/upgrade_to_continuous_delivery.sh

```diff
@@ -84,8 +84,6 @@ function hash_operator() {
         "ibm-cert-manager-operator") eval "$2='3.8.0'";;
         "ibm-mongodb-operator") eval "$2='1.2.0'";;
         "ibm-iam-operator") eval "$2='3.8.0'";;
-        "ibm-monitoring-exporters-operator") eval "$2='1.9.0'";;
-        "ibm-monitoring-prometheus-operator-ext") eval "$2='1.9.0'";;
```

Comment:
This is for the upgrade from EUS to continuous delivery like 3.7 or 3.8.

It is used to check whether the operator has been upgraded to version 3.6. So we should not delete it.

Reply:
Thanks, I will revert them.